### PR TITLE
Do not render link for deleted user

### PR DESCRIPTION
**Pull request by matthew-white**
See getodk/central#127.

### CODE REVIEW

File: src/components/audit/row.vue [hunk 1]
```diff
@@ -19,19 +19,22 @@ except according to the terms contained in the LICENSE file.
       </template>
     </td>
     <td class="initiator">
-      <actor-link v-if="audit.actor != null" :actor="audit.actor"/>
+      <template v-if="audit.actor != null">
+        <span v-if="audit.actor.deletedAt != null" class="icon-trash" :title="$t('deletedMessage')"></span>
+        <actor-link :actor="audit.actor"/>
```
Comment (matthew-white) [inline src/components/audit/row.vue:24]: The white space between the `<span>` and the `<actor-link>` is automatically trimmed, so to add space between them, I use `$margin-right-icon` below. (I often do that with icons.) I also tweaked the Target column to do something similar.

File: src/components/audit/row.vue [hunk 2]
```diff
@@ -19,19 +19,22 @@ except according to the terms contained in the LICENSE file.
...
           :title="target.title">
           {{ target.title }}
         </router-link>
-        <span v-else :title="target.title">
+        <template v-else>
           <span v-if="target.deleted" class="icon-trash" :title="$t('deletedMessage')"></span>
-          <span v-if="target.purged" class="icon-trash" :title="$t('purgedMessage')"></span>
-          {{ target.title }}
-        </span>
+          <span v-else-if="target.purged" class="icon-trash" :title="$t('purgedMessage')"></span>
```
Comment (matthew-white) [inline src/components/audit/row.vue:35]: Just a little optimization that I noticed was possible

File: test/data/users.js [hunk 3]
```diff
@@ -33,7 +34,8 @@ export const extendedUsers = dataStore({
     createdAt: createdAt != null
       ? createdAt
       : (inPast ? fakePastDate([lastCreatedAt]) : new Date().toISOString()),
-    updated: null
+    updatedAt: null,
```
Comment (matthew-white) [inline test/data/users.js:37]: Fixing a mistake that I noticed

**Comment by matthew-white** [comment]
@ktuite, I think this can be reviewed async!

**Review by ktuite — APPROVED** [review]
Looks good to me and I've tried it out locally!

**Pull request merged**